packages/manager/apps/web/client/app/hosting/multisite/statistics/hosting-multisite-statistics.component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './hosting-multisite-statistics.controller';
+import template from './hosting-multisite-statistics.html';
+
+export default {
+  bindings: {
+    chartJsFactory: '<',
```

Comment:
Injected through routing ?

Reply:
No it comes from an injection here, and then passed to the component : https://github.com/ovh/manager/pull/4738/files/c557b86b745ea252fa2781217114e9a94a9a3062#diff-4a8532ebd5034b41690cf07d70b1caa09fc4a0e4d01f5787b56495d10bb7b02fR43